README.md

```diff
@@ -83,7 +83,22 @@ When deploying the multi cluster traffic controller using the make targets the f
     ```sh
     (export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build install run
     ```
-## 3. Run the ingress agent locally
+
+## 3. Deploy the agent
+1. Update the secret in `config/agent/secret.yaml` with the correct credentials for the control plane. (**N.B.** The server should remain `https://mctc-control-plane-control-plane:6443`)
+
+1. Build the agent image and load it into the workload cluster
+    ```sh
+    export KUBECONFIG=./hack/kubeconfigs/mctc-workload-1.kubeconfig
```

Comment:
Is there any difference between this and `kubectl config use-context kind-mctc-workload-1`? Just out of curiosity

Reply:
just that you can set different values in different terminals. Using `kubectl config use-context` changes the context in all terminals (I like to have one terminal connected to the control-plane and one connected to the workload cluster)

Reply:
Oh yes I hadn't thought of that! That's definitely better